src/Uno.UI.RemoteControl.VS/IDEChannel/IDEChannelClient.cs

```diff
@@ -3,11 +3,16 @@
 using System.Linq;
 using System.Threading;
 using System.Threading.Tasks;
+using EnvDTE;
+using Microsoft.VisualStudio.Imaging;
+using Microsoft.VisualStudio.Shell;
+using Microsoft.VisualStudio.Shell.Interop;
```

Comment:
Same here, I think we don't need all those namespaces?

Reply:
Done. Removed. https://github.com/unoplatform/uno/commit/da8d2feb4472187808112622109a6ec1819a1004